pkg/machines/components/vmSnapshotsTab.css

```diff
@@ -0,0 +1,19 @@
+.no-snapshots {
+    text-align: center;
+    line-height: 3.0rem
+}
+
+.snap-greyed-out {
+    color: var(--pf-global--palette--black-600);
+}
+
+.tooltip-circle {
+    padding: .5rem
+}
+
+.snap-current {
+    border-left: 3px solid var(--pf-global--primary-color--100);
+    background-color: var(--pf-global--BackgroundColor--200)
```

Comment:
I was looking around at what other projects do for this.
* Virt-manager: checkmark (✔️)
* GNOME Boxes: square is different color [◼️]
* OpenShift: [Nothing particular](http://openshift.github.io/openshift-origin-design/designs/virtualization/4.3/snapshots/)
* oVirt: [Change date and description](https://3.bp.blogspot.com/-U8QMcx9JFA4/XBhjV5hFI1I/AAAAAAAAAgw/eAPNOt_hWh087A9Ns2vrRsYWnUcitpkUgCLcBGAs/s1600/Selection_999%2528836%2529.png)

I kind of like the oVirt approach, but just a small visual indicator like this works too.

Reply:
The selected visual state from PF4 is for _interactive_ selections (like nav, tab, or a selected row in a table), not a way to show something is active or current. And it needs to be obvious not just to screen readers (with ARIA) but also visually in the browser. The blue line "doesn't cut it" (is not good enough), in other words. 

The others aren't really obvious either (from your description, at least). It's nice to have a visual indicator, but if it is relying only on color or is just an undefined glyph, it isn't enough. (That could be changed with a tooltip for the icon and appropriate ARIA on the active element.)

Reply:
> (That could be changed with a tooltip for the icon and appropriate ARIA on the active element.)

What icon could be best for highlighting active/current snapshot?

Reply:
PF4 happens to use `fa-check-circle` in a label their example for "running" on the icons page:

![status icons](https://www.patternfly.org/v4/img/ffc62ca48f6c8d914f97acff46e8e893-sm-icons-1.png)
(_These look large here, since I guess it's a 2× screenshot. They should not look so large when used in Cockpit, of course._)

- Icons page: https://www.patternfly.org/v4/design-guidelines/styles/icons
- Label component: https://www.patternfly.org/v4/documentation/react/components/label

This mirrors my suggestion of using an "active" badge (I really meant label, but labels are newer to PF and are extremely similar to badges anyway).

If space is of concern, we could just use the icon alone with a tooltip (and use ARIA on the selected item, as I suggested above).

If we're using a label, it should go _after_ the text; if an icon, it should go into its own column preceding everything (with the others only having empty space where the icon would be).

FWIW: Regardless of implementation, the date and description could also change.

Reply:
So something like this?
![Screenshot from 2020-07-01 12-57-16](https://user-images.githubusercontent.com/209895/86236605-a7a22300-bb9a-11ea-8d37-120b1df14d9c.png)

Reply:
Sure, that works for me! Thanks! (Once it's "shut off".)

Reply:
Or maybe like this?

![Screenshot from 2020-07-01 12-51-13](https://user-images.githubusercontent.com/209895/86237020-4d559200-bb9b-11ea-9f63-57af6755d7b4.png)

Reply:
I like first mockup. Getting rid of Time fully and replacing it with icon is not nice, I think user should be able to see time of creation at all time (even if they decide to not include that in snapshotname)..

Reply:
Sure, let's go for the first one!